testkit/src/checkpoint_db.rs

```diff
@@ -73,31 +74,50 @@ pub struct CheckpointDbHandler<T> {
 }
 
 impl<T: Database> CheckpointDbHandler<T> {
+    /// Set a checkpoint for future rollback
+    ///
+    /// # Panics
+    ///
+    /// - Panics if another checkpoint was set before and have not been rolled back to.
+    pub fn checkpoint(&self) {
+        self.inner
+            .write()
+            .expect("Cannot lock CheckpointDb for checkpoint")
+            .checkpoint();
+    }
+
     /// Rolls back this database by undoing the latest `count` `merge()` operations.
     ///
     /// # Panics
     ///
-    /// - Panics if more operations are attempted to be reverted than the number of operations
-    ///   in the DB journal.
-    pub fn rollback(&self, count: usize) {
+    /// - Panics if there is no checkpoint to rollback to.
+    pub fn rollback(&self) {
         self.inner
             .write()
             .expect("Cannot lock CheckpointDb for rollback")
-            .rollback(count);
+            .rollback();
     }
 }
 
 #[derive(Debug)]
+// Journal is used here to track patches that need to be applied (in order
+// that they appear) to db for it to become what it was on checkpoint.
```

Comment:
`Journal is used to track patches that need to be applied (in the order of their appearance) to the database in order to restore its state to a checkpoint.`

Reply:
I'd prefer `in order that they appear` or maybe `in order in which they appear`, but pretty much agree with everything else. Thanks for helping with my terrible English grammar!

Reply:
Okay, but please add `the` preposition before `order`. `In order` and `in the order` have different meanings.